scripts/pyro_config_server.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python
+"""
+Script to start the config server. Make sure the Pyro name server is already
```

Comment:
If it starts it, then how about renaming to `start_pyro_config_server.py` or even just `start_config_server.py`